src/mcpbr/cli.py

```
@@ -1531,6 +1536,161 @@ def benchmarks() -> None:
+    "--filter-category",
+    multiple=True,
+    help="Filter by language or repo substring.",
+)
+@click.option(
+    "--timeout",
+    type=int,
+    default=300,
+    help="Timeout per test in seconds (default: 300).",
+)
+def preflight(
+    config_path: str | None,
```

Ruff (0.15.2)

[warning] Unused function argument: `config_path`

(ARG001)